docs/sql-ref-syntax-dml-insert-into.md

```diff
@@ -40,8 +40,8 @@ INSERT INTO [ TABLE ] table_identifier [ partition_spec ] [ ( column_list ) ]
 
 * **partition_spec**
 
-    An optional parameter that specifies a comma-separated list of key and value pairs
-    for partitions.
+    An optional parameter that specifies a comma separated list of key and value pairs
```

Comment:
nit: why did you changed `comma-separated` => `comma separated`?

Reply:
Done